assets/css/lifterlms.css

```diff
@@ -2379,7 +2379,7 @@ ul.llms-payment-gateways .llms-payment-gateway-error {
   padding: 0;
 }
 .llms-form-field.type-radio input,
-.llms-form-field.type-radio label, .llms-form-field.type-checkbox input,
+.llms-form-field.type-radio label.llms-radio, .llms-form-field.type-checkbox input,
```

Comment:
CSS changes made directly to the compiled `.css` files in the assets directory will be overwritten during build steps.

Instead, make css changes to the .scss file found here: https://github.com/gocodebox/lifterlms/blob/master/_private/scss/_includes/_llms-form-field.scss

After updating you can rebuild the compiled css files via `gulp dist`

See [installing for development](https://github.com/gocodebox/lifterlms#installing-for-development) in the readme.

Reply:
As an aside I understand that our SCSS is a little bit of a cluster and I was going to merge this in manually but I found the duplicated checkbox issue while testing

Reply:
you know better then me the code :-)

Reply:
If you can figure out the checkbox issue reliably and want me to deal with the SCSS I can manage :-D

Reply:
added the break, for scss you know better me what to do. I don't have time now to work on that.